docs/imagecustomizer/api/cli.md

```diff
@@ -86,13 +86,27 @@ Can be one of:
 
   The RPMs may either be in the directory itself or any subdirectories.
 
+  GPG signature checking is disabled for local directories.
+  If you wish to enable GPG signature checking, then use a repo file instead and set the
+  `gpgkey` field within the repo file.
+
 - `*.repo` file path: A path to a RPM repo definition file.
 
   The file name extension must be `.repo`.
 
-  Note: This file is not installed in the image during customization.
-  If that is also needed, then use `AdditionalFiles` to place the repo file within
+  If the repo file's `baseurl` or `gpgkey` fields contain a `file://` URL, then the
```

Comment:
nit: should we include a sample repo file?

Reply:
The bullet point is getting pretty long as it is. If I wanted to add example repo files, I'd probably need to reorganize the docs. Maybe create a concepts page for RPM repos or something.